qtacrylicwidget.cpp

```diff
@@ -171,3 +171,10 @@ void QtAcrylicWidget::paintEvent(QPaintEvent *event)
     }
     QWidget::paintEvent(event);
 }
+
+void QtAcrylicWidget::changeEvent(QEvent* event)
```

Comment:
Why was this needed for QtWidget? It dones't have the layout issue as QMainWindow.